app/src/main/java/org/gdg/frisbee/android/api/SynchronousCallAdapterFactory.java

```diff
@@ -0,0 +1,47 @@
+package org.gdg.frisbee.android.api;
+
+import java.io.IOException;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+
+import retrofit2.Call;
+import retrofit2.CallAdapter;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import timber.log.Timber;
+
+public class SynchronousCallAdapterFactory extends CallAdapter.Factory {
```

Comment:
It won't add much difference. With Call class, you will just add `execute` method and it will do the same. That's whats you are already doing with `getImageInfo` actually.